### ISSUE TITLE
Noto Serif LGC support for Nuer language appears broken

### ISSUE THREAD

**Issue by davelab6** [google/fonts#4835]
@andjc wrote me

> Go to the Noto Serif specimen page, select Africa, then select under language dropdown Nuer to get the Nuer specimen. 
> 
> (Screen Shots taken in Edge, but similar effect in Chrome and Firefox.)
> 
> Two different effects are seen. 
> 
> In type specimen, you have font substitution for open-e and open-o .
> 
> ![Screenshot_20220626-182710](https://user-images.githubusercontent.com/261579/175840188-801c3e96-9c46-4e37-b37b-e701127e9010.png)
> 
> In the waterfall you get the .notdef glyph instead
> 
> ![Screenshot_20220626-182608](https://user-images.githubusercontent.com/261579/175840158-11bb2173-9a80-406f-a434-bc2acb9fd464.png)
> 
> Nuer, like many African languages, needs one of the alternative glyphs for Eng, which unless the font is available is only accessible via aalt feature, a far from ideal solution for web fonts, a good solution for InDesign.
> 
> i assume that the type specimens are using GF version of the fonts, so are being subsetted, since the characters are in the font upstream 

We can update the Extended Latin subset definition to add missing characters to the web font, if that's indeed the issue as I guess :)

> But I would suggest next time the Noto font pages are redesigned you explicitly detail for each language what opentype features they need to activate to get the glyphs needed for the language if not supported by default glyphs. Also worth noting that aalt works best with per character markup, if you are using a wide range of languages 

Agreed, the way we handle and surface OpenType Features has lots of room for improvement :)

**Comment by andjc** [comment]
The following characters are impacted: U+0254, ‎U+025B, U+0263, ‎U+0308, and U+0331.

Also worth noting the uppercase equivalents are not impacted by this.

**Comment by simoncozens** [comment]
These characters should probably be added to the Latin extended nam file. Is that the right approach, @nathan-williams?

**Comment by andjc** [comment]
@simoncozens  does it also address which Eng glyph is used?

**Comment by simoncozens** [comment]
No. The selection of glyphs is a font-level issue; the issue I was dealing with here is that certain codepoints were being subsetted away from the font before we even got to glyph selection. Eng selection is being tracked here: https://github.com/notofonts/latin-greek-cyrillic/issues/161
